doc/nrf/ug_thingy53.rst

```diff
@@ -0,0 +1,258 @@
+|                                |``thingy53_nrf5340_cpuapp_ns`` for the non-secure version |
++--------------------------------+----------------------------------------------------------+
+| nRF5340 SoC - Network core     |``thingy91_nrf52840``                                     |
++--------------------------------+----------------------------------------------------------+
+
+The table below shows the different types of build files that are generated and the different scenarios in which they are used:
+
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| File                        | File format                          | Programming scenario                                               |
++=============================+======================================+====================================================================+
+| :file:`merged.hex`          | Full image, HEX format               | Using an external debug probe and nRF Connect Programmer           |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`app_signed.hex`      | MCUboot compatible image, HEX format | Using the built-in bootloader and nRF Connect Programmer           |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`app_update.bin`      | MCUboot compatible image, HEX format | Using the built-in bootloader and the mcumgr command line tool     |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+| :file:`dfu_application.zip` | MCUboot compatible image, binary     | Using nRF Programmer for Android and iOS or nRF Connect Programmer |
++-----------------------------+--------------------------------------+--------------------------------------------------------------------+
+
+There are multiple methods of programming a sample or application onto a Thingy:53.
+You can choose the method based on the availability or absence of an external debug probe to program.
+
+.. note::
+
+   If you do not have an external debug probe available to program the Thingy:53, you can directly program the device over USB using the built-in MCUboot bootloader and nRF Connect Programmer.
+   In this scenario, use the :file:`dfu_application.zip` file.
+
+.. _thingy53_build_pgm_segger:
+
+Building and programming using SEGGER Embedded Studio
+*****************************************************
+
+.. include:: gs_programming.rst
+   :start-after: build_SES_projimport_open_start
+   :end-before: build_SES_projimport_open_end
+
+..
+
+   .. figure:: images/ses_thingy53_configuration.png
+      :alt: Opening the Asset tracker application
```

Comment:
`Opening the Asset tracker application`
This app will not work on thingy53, so we should not reference it

Reply:
In my word draft this is nRF Machine Learning. Dont know if the draft has not been synced to OneDrive when the PR was created.

Reply:
The text was nRF Machine Learning, but the image showed Thingy:91 with the Asset Tracker app. I initially had all text match the image, but changed the image today but apparently missed one part of the text.